packages/farcaster-js-hub-rest/src/utils.ts

```diff
@@ -0,0 +1,13 @@
+import { Eip712Signer, EthersEip712Signer, NobleEd25519Signer } from '@farcaster/core';
+import { hexToBytes } from '@noble/hashes/utils';
+import { Wallet } from 'ethers';
+
+export function hexToSigner(signerHex: string): NobleEd25519Signer {
+  const privateKeyBytes = hexToBytes(signerHex.slice(2));
+  return new NobleEd25519Signer(privateKeyBytes);
+}
+
+export function eip712SignerFromPhrase(mnemonic: string): Eip712Signer {
```

Comment:
Is this used?

Reply:
It's used in submitVerification to get the address of the account to verify and to sign the verification claim

```
const verificationSigner = eip712SignerFromPhrase(verification.verifiedAddressMnemonic);
const addressBytes = await verificationSigner.getSignerKey();
const ethSignResult = await verificationSigner.signVerificationEthAddressClaim(claim.value);
```

Reply:
https://github.com/standard-crypto/farcaster-js/blob/hub-rest-submit-message/packages/farcaster-js-hub-rest/src/hubRestApiClient.ts#L410